addons/logistics_wirecutter/script_component.hpp

```diff
@@ -59,5 +59,6 @@
 
 #define HAS_WIRECUTTER(unit) (\
     "ACE_wirecutter" in ([ARR_6(unit, false, true, true, true, false)] call CBA_fnc_uniqueUnitItems) \
-    || {backpack unit isKindOf "rhs_assault_umbts_engineer"} \
+    || {1 == getNumber (configFile >> "CfgVehicles" >> (backpack unit) >> QGVAR(hasWirecutter))} \
+    || {1 == getNumber (configFile >> "CfgWeapons" >> (vest unit) >> QGVAR(hasWirecutter))} \
```

Comment:
Vests are CfgVehicles.

Reply:
They are Items/CfgWeapons.
There is a `vestContainer` command, that returns the vest "object"
but `typeOf vestContainer player` = `"Supply140"`

Reply:
[Vests are CfgWeapons](https://community.bistudio.com/wiki/Arma_3_Characters_And_Gear_Encoding_Guide#Vest_configuration)
[Vest ground holders are CfgVehicles](https://community.bistudio.com/wiki/Arma_3_Characters_And_Gear_Encoding_Guide#Vest_ground_holder)